layouts/partials/code.html

```diff
@@ -3,7 +3,7 @@
     {{ $lang = "cql" }}
 {{ end }}
 
-<div class="code-mode-{{ default "light" .mode }} {{ if .chrome }} code-chrome {{ end }} code-{{ .lang }}">
+<div class="code-hero code-mode-{{ default "light" .mode }} {{ if .chrome }} code-chrome {{ end }} code-{{ .lang }}">
```

Comment:
I added `code-hero` so I could use that in a selector to exclude the copy button from these. Though, I could see wanting to provide it for some uses. As a follow-up, we could add another option to enable it for this partial. (Would require some minor style updates for dark mode).

Reply:
Having `hero` in the classname bugs me a teeny bit (since there are lots of non-hero places where these are used), but it's only here, so definitely a nit. Maybe `.no-copy` (like `.no-select`), though? Up to you, just thinking aloud.

And yes, it'd be great to have these under Products as well! Could just surface them on hover, too.